infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -56,16 +60,50 @@ def run_shell_command(command, env_vars):
 ###################
 
 
-def make_backup_fpath():
-    return BACKUP_DIR / datetime.now().strftime(FILENAME_DATETIME_FORMAT)
+def make_object_name():
+    return datetime.now().strftime(FILENAME_DATETIME_FORMAT)
 
 
-def backup_database(db_backup_gzip_fpath):
+def make_backup_fpath(object_name):
```

Comment:
Could you add docstrings to these? Sorry to be annoying about this. We have a standard style guide here: https://www.notion.so/Development-Practices-accc91a1cf3a4081ab42c0af24dd1c8a#153ffd8fe49f464c8e17cb1211a5c1ee

Actually, I'm hoping to automate a `pre-commit` hook that reminds you if you haven't added docstrings because it's really annoying when reviewers pick on you for that.

Reply:
shame on me lol. yes of course.

Reply:
Ah, part of this is adding type annotations like

```python
def func(some_param: some_type) -> some_return_type:
    """docstring"""
    ...
```

Reply:
I added docstrings and static typing to ALL of the functions in the file